jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -278,24 +282,41 @@ public void onFrame(Frame frame, Callback callback)
             dataType = OpCode.UNDEFINED;
     }
 
+    public void onClose(Frame frame, Callback callback)
```

Comment:
why public?  surely protected?

Reply:
Yes I guess this could be protected, all the other methods `onText`, `onPong`, etc.. are public, these methods all already existed. Should I change them all to protected?